dishes/peruvian/condimentos/README.md

```
@@ -0,0 +1,14 @@
+# Condimentos y Salsas Peruanas
+
+Los condimentos y salsas son esenciales en la cocina peruana, aportando sabor, color y complejidad a los platos.
+
+## Ejemplos de condimentos y salsas
```

markdownlint-cli2 (0.17.2)

Headings should be surrounded by blank lines
Expected: 1; Actual: 0; Below

(MD022, blanks-around-headings)